PolysyncLidarTranslator/README.md

```diff
@@ -0,0 +1,83 @@
+To properly build this example with the generated PolySync messages in ROS, you will first need to set up your catkin workspace with the ros_bridge.
```

Comment:
Please add a one or two sentence introduction to what the example does, and the value it provides. Listing other use cases on how to extend the example would add a lot of value.